submissions/evgeniy241984/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,102 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+"use strict";
+
+class Animals {
+  constructor(gender, name, saying) {
+    this.gender = gender;
+    this.name = name;
+    this.saying = saying;
```

Comment:
Every inhabitant has species attribute. It seems the root class to be a store for this property.